apps/storybook/src/stories/decorators/toaster.tsx

```diff
@@ -1,6 +1,6 @@
-import { Toaster } from "#components/ui/toaster";
+import { Toaster } from "#components/ui/sonner";
 
-export const withToaster = () => {
+export const withSonner = () => {
```

Comment:
lets stick with `withToaster` (align file name & semantics)

Reply:
also lets fix test and remove `use-toast` import
https://github.com/fractaldotbox/ddevkit/actions/runs/14430370481/job/40464423180?pr=139#step:4:36